client/me/help/help-contact/index.jsx

```diff
@@ -76,10 +76,8 @@ const SUPPORT_LIVECHAT = 'SUPPORT_LIVECHAT';
 const SUPPORT_TICKET = 'SUPPORT_TICKET';
```

Comment:
Kind of unrelated but maybe we can quick fix it here, `state/current-user/selectors` is imported twice.

Reply:
Good catch! Fixed in https://github.com/Automattic/wp-calypso/pull/20814/commits/a09a21ebf084dd97259755b57aba456b0aa68961